runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.tags.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useState, useRef, useLayoutEffect } from 'react';
+
+import If from '#src-app/components/utils/If';
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import {
+    StyledTypography,
+    TagBox,
+    DividerBox,
+    DividerAction,
+    DividerLine,
+    StaticLine,
+    Container,
+    StyledExpandIcon,
+    StyledChipName
+} from './ProcessTile.tags.styles';
+import { ProcessTileTagsProps } from './ProcessTile.types';
+
+const TAGS_CONTAINER_MARGIN_VALUE = 85;
+const TAG_RIGHT_MARGIN_VALUE = 8;
+
+const ProcessTileTags: FC<ProcessTileTagsProps> = ({ tags }) => {
+    const { translate } = useTranslations();
+
+    const [areTagsExpanded, setAreTagsExpanded] = useState<boolean>(false);
+    const [areTagsFittingLine, setAreTagsFittingLine] = useState<boolean>(false);
```

Comment:
Don't use `are` in boolean variable naming. It's better to use another word for it:
`[isEveryTagExpanded, setEveryTagExpanded]`
`[isTagsListMultiLine, setTagsListMultiLine]`